docs/source/gettingstarted.rst

```diff
@@ -76,22 +76,21 @@ same folder that you're currently in with the following content:
     SELECT a+b  AS foo,
     c AS bar from my_table
 
-You can then run :code:`sqlfluff lint test.sql` to lint this file.
+You can then run :code:`sqlfluff lint test.sql --dialect ansi` to lint this file.
```

Comment:
```suggestion
You can then run :code:`sqlfluff lint test.sql --dialect ansi` to lint this
file.
```

Reply:
Does this imply that my environment (VSCode) wasn't set to wrap at 80 characters?

Reply:
Wasn't aware you could set line limits in VSCode.